systemverilog-plugin/UhdmAst.cc

```diff
@@ -1894,7 +1895,6 @@ void UhdmAst::process_design()
                               shared.top_nodes[node->str] = node;
                           }
                       });
-    visit_one_to_many({vpiParameter, vpiParamAssign}, obj_h, [&](AST::AstNode *node) {});
```

Comment:
Isn't it needed for processing children of parameters and parameter assignments on file level?

Reply:
The problem was that it was never added to the design node anyway (so it didn't work), but you are right, I've added support for this instead. New yosys-systemverilog run: https://github.com/antmicro/yosys-systemverilog/actions/runs/4913504824

Reply:
Ok, nice!